apps/indexer/lib/indexer/supervisor.ex

```diff
@@ -198,8 +196,19 @@ defmodule Indexer.Supervisor do
         fetchers_with_amb_bridge_mediators
       end
 
+    fetcher_with_epoch_rewards =
+      if System.get_env("DISPLAY_REWARDS") === "true" do
```

Comment:
Also the syntax looks different to other places where env variables are fetched, what's the difference with this one?

Reply:
Only reason is that I wanted to have exactly same check in both places: https://github.com/celo-org/blockscout/blob/master/apps/block_scout_web/lib/block_scout_web/templates/address/_tabs.html.eex#L70

and it's temporary anyway to exclude the possibility of this fetcher having such impact on the app